raceid3/raceid3_workflow-test.yml

```diff
@@ -0,0 +1,30 @@
+- doc: Test RaceID workflow in Galaxy Europe
+  job:
+    count_matrix:
+      class: File
+      location: use.intestinal
```

Comment:
Should that not be a URL pointing to a real matrix?

Reply:
The file is literally a 1x1 'matrix' with the word "test". It is a signature file of sorts, that tells the tool to  load the internal example data. 

I could upload the raw example data to zenodo (~30MB) if that would be better.

Reply:
@mtekman please do not use this `load internal data`, this is a huge hack. A matrix that can be potentially used in the training material would make a lot of sense here. We want to test a real workflow here and not the fake hack, imho.

Reply:
@bgruening fair enough, will upload the dataset to zenodo

Reply:
Thanks.

Reply:
@bgruening tests run good, any further changes?